models/schema.yml

```diff
@@ -279,3 +281,171 @@ models:
         description: "The number of units in inventory of the product variant"
         tests:
           - not_null
+
+  - name: shopify__refunds
+    description: "A table storing information regarding order refunds"
+    columns:
+      - name: refund_id
+        description: "The unique identifier of the refund"
+        tests:
+          - unique
+          - not_null
+      - name: order_id
+        description: "The unique identifier of the order"
+        tests:
+          - not_null
+          - relationships:
+              to: ref('shopify__orders')
+              field: order_id
+      - name: user_id
+        description: "The unique identifier of the user who created the refund"
+        tests:
+          - not_null
+      - name: total_price
```

Comment:
surely some refunds wont be a full refund but a partial refunds?

Reply:
ok, changed to sum(order_refunds_refund_line_items.subtotal)

Reply:
im confused it still says total price?

Reply:
these 2 won't show up here. This is outdated. You need to look at the source code